spec/unit/installer/xcode/pods_project_generator_spec.rb

```diff
@@ -282,7 +284,7 @@ def test_extension_target(symbol_type)
               mock_native_target.expects(:add_dependency).with(test_dependent_native_target).never
               mock_native_target.expects(:add_dependency).with(mock_native_target).never
 
-              mock_test_native_target.expects(:add_dependency).with(dependent_native_target).never
```

Comment:
this changed because we now add a dependency to the transitive targets as well for test native targets.